ServiceMonitoringFunction/tst/com.amazon.vas.serviceordermonitoringlambda/util/DefaultModelBuilders.java

```diff
@@ -0,0 +1,95 @@
+package util;
+
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsInput;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsInputBO;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsOutput;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsOutputBO;
+import com.amazon.vas.serviceordermonitoringlambda.model.JobAggregatedMetricsBO;
+import com.amazon.vas.serviceordermonitoringlambda.model.JobDetailsBO;
+import com.google.gson.Gson;
+import org.json.simple.JSONObject;
+
+import java.util.*;
+
+import static util.TestConstants.*;
+
+public class DefaultModelBuilders {
+
+    private final int numberOfTestJobs = 100;
+    private final long baseTime = 1592800200;
+    private final long timeBias = 3600;
+
+    public GetJobMetricsInputBO buildGetJobMetricsInputBO(String filteringKey){
+        GetJobMetricsInputBO getJobMetricsInputBO = GetJobMetricsInputBO.builder().city(null)
+                .merchantId(null).serviceCategory(null)
+                .groupingCriteria(new ArrayList<>(Arrays.asList(filteringKey, SLOT_START_TIME))).build();
+        return getJobMetricsInputBO;
+    }
+    public GetJobMetricsInput buildGetJobMetricsInput(String filteringKey){
+        GetJobMetricsInput getJobMetricsInput = GetJobMetricsInput.builder().city(null)
+                .merchantId(null).serviceCategory(null)
+                .groupingCriteria(new ArrayList<>(Arrays.asList(filteringKey, SLOT_START_TIME))).build();
+        return getJobMetricsInput;
+    }
+
+    public ArrayList<JobDetailsBO> buildJobDetailsList() {
```

Comment:
you should use base type for colelction(List or Map etc) while declaring / returning type from function. This applicable across source and tests.